src/packages.jl

```diff
@@ -32,19 +32,22 @@ the package banner is not printed.
 If `install` is set to `true` and the required GAP package is not yet
 installed then [`install`](@ref) is called first, in order to install
 the newest released version of the package.
+
+If `quiet` is set to `true` then package banners are shown for all packages
+being loaded. It is also passed on to [`install`](@ref).
 """
-function load(spec::String, version::String = ""; install = false)
+function load(spec::String, version::String = ""; install::Bool = false, quiet::Bool = true)
     # Try to load the package.
     gspec = GapObj(spec)
     gversion = GapObj(version)
-    loaded = Globals.LoadPackage(gspec, gversion, false)
+    loaded = Globals.LoadPackage(gspec, gversion, !quiet)
     if loaded == true
         return true
     elseif install == true
         # Try to install the package, without showing messages.
-        if Packages.install(spec; interactive = false, quiet = true)
+        if Packages.install(spec; interactive = false, quiet)
```

Comment:
`Packages.install(spec; interactive = false, quiet = quiet)`?

Reply:
That's equivalent; Julia 1.6 allows replacing kwargs of the form `foo = foo` by just `foo`

Reply:
O.k.
(I do not like this syntax. The left hand side of `foo = foo` is a key, the right hand side is a variable.
When one copies such a code line into another context and adjusts the variable name then one gets into trouble.)